.github/workflows/ci.yml

```
@@ -57,6 +60,9 @@ jobs:
       run-unit: true
       run-component: true
       run-integration: false
+    permissions:
+      contents: read
+      actions: write  #? For node caching
```

YAMLlint (1.37.1)

[warning] missing starting space in comment

(comments)